plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java

```diff
@@ -54,6 +56,125 @@ public void destroy()
         TestingPhoenixServer.shutDown();
     }
 
+    @Override
+    protected boolean supportsDelete()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsViews()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsArrays()
+    {
+        return true;
+    }
+
+    @Override
+    protected boolean supportsCommentOnTable()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsCommentOnColumn()
+    {
+        return false;
+    }
+
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        throw new SkipException("Phoenix connector does not support column default values");
+    }
+
+    @Override
+    public void testRenameTable()
+    {
+        // Phoenix does not support renaming tables
```

Comment:
`SkipException`?

Reply:
Actially it should be `assertThatThrownBy(super.testRenameTable()); throw new SkipException()`
But it is preexistent.